docs/NetCore_GettingStarted.md

```diff
@@ -13,9 +13,10 @@
 |Branch| SDK | Description|
 |--------|--------|--------|
 |[2.9.x](https://github.com/dotnet/roslyn-analyzers/tree/2.9.x)| Does not ship in the .NET SDK | A special branch compatible with Visual Studio 2017 where security analyzers are shipped from.
```

Comment:
@mavasani Is this still relevant?

Reply:
Yes, it is. The branch is quite old and only present in case the security folks want a hot fix.